src/parser/warrior/protection/modules/talents/AngerCD.js

```diff
@@ -117,7 +122,7 @@ class AngerCD extends Analyzer {
     when(this.suggestionThresholdsLastStand).addSuggestion((suggest, actual, recommended) => {
       return suggest(
         <>
-          Try to cast <SpellLink id={SPELLS.LAST_STAND.id} /> more often.
+          Remember to cast <SpellLink id={SPELLS.LAST_STAND.id} /> more frequently. If you have <SpellLink id={SPELLS.BOLSTER_TALENT.id} /> talented you can use this to fill gaps between your <SpellLink id={SPELLS.SHIELD_BLOCK.id} />.
```

Comment:
Between my Shield Block what? I'm assuming you mean to have some mitigation for when Shield Block drops, but it isn't obvious from this. Consider rewording.